moonsense/client.py

```diff
@@ -93,10 +99,13 @@ def whoami(self) -> TokenSelfResponse:
         r = requests.get(endpoint, **self._headers)
         return json_format.Parse(r.text, TokenSelfResponse(), ignore_unknown_fields=True)
 
-    def list_sessions(self, labels: List[str] = None, client_session_group_id: str = None) -> Iterable[Session]:
+    def list_sessions(self, labels: List[str] = None, client_session_group_id: str = None, platforms: List[Platform] = None) -> Iterable[Session]:
         """
         List sessions for the current project
 
+        :param labels: A list of labels to match.
+        :param client_session_group_id: The client session group id to match
+        :param platforms: The list of 'Platform's to match.
```

Comment:
Add a note that using `None` returns all platfforms?

Reply:
Done